docker-for-windows/troubleshoot.md

```diff
@@ -307,10 +340,9 @@ Here are some steps to take if you encounter similar problems:
     > Read the whole page before running any scripts, especially warnings about `-ForceDeleteAllSwitches`.
     > {: .warning}
 
-### Windows containers and Windows Server 2016
```

Comment:
I think if we want to change this, do it in another PR, unrelated to Kubernetes.
Here we need to keep a mention of what users can expect on windows server. OK for your update on Windows 10. On windows server, we should mention Docker Desktop is not supported on windows Server and users can use docker EE basic that comes with Windows Server 2016. 
See Docker Desktop https://docs.docker.com/install/windows/docker-ee/#about-docker-ee-containers-and-windows-server
and http://success.docker.com/article/where-to-get-help-with-windows

Reply:
I took it off because in the text it only referred to an old link with With information on both.  But you are right I'll add a paragraph to link it to the windows server docs :D